### PR TITLE
Update Matching Interface Flash Message

### PR DESCRIPTION
**Pull request by paulgain**
## Update of the Flash message
Provide more information to the user once that have matched a company. 
### Before
before
### After
after
## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or Acceptance)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: test/functional/cypress/specs/companies/match-company-spec.js [hunk 1]
```diff
@@ -130,7 +130,8 @@ describe('Match a company', () => {
     it('displays the "Company record update request sent" flash message', () => {
       cy.get(localHeader().flash).should(
         'contain.text',
```
Comment (rafenden) [inline test/functional/cypress/specs/companies/match-company-spec.js]: We could be more strict here and use `have.text` instead.

Comment (paulgain) [reply test/functional/cypress/specs/companies/match-company-spec.js]: Initially I did try that, however, the test failed due to the new line.

```html
<li class="c-message c-message--success">
  Your request to match this company record has been sent.
  <br>
  In the near future the Support Team will update this company record and enable automatic updates for future changes to 
  the company information.
  <a class="c-message__close" href="#">Dismiss</a>
</li>
```

**Comment by web-bert** [comment]
@paulgain we had a similar request to put additional text in the flash message. Should we update it to have a heading and body text instead? The all bold text seems a bit too much...

**Comment by paulgain** [comment]
> @paulgain we had a similar request to put additional text in the flash message. Should we update it to have a heading and body text instead? The all bold text seems a bit too much...

@web-bert Good idea, I've just spoken to Roger and he also agrees - let's do it!

File: src/apps/companies/apps/match-company/controllers.js [hunk 2]
```diff
@@ -151,7 +151,11 @@ async function submitMatchRequest(req, res) {
     const ticket = createMatchRequestMessage(req, res)
     const result = await postToZenDesk(ticket)
 
-    req.flash('success', 'Company record update request sent')
+    req.flash(
+      'success',
+      `Your request to match this company record has been sent.
+       In the near future the Support Team will update this company record and enable automatic updates for future changes to the company information.`
```
Comment (peterhudec) [inline src/apps/companies/apps/match-company/controllers.js]: I would line break it so that it fits into 80 chractaers.

Comment (paulgain) [reply src/apps/companies/apps/match-company/controllers.js]: 👍

**Pull request merged**